webstepper/src/CodeDisplay.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { a11yLight } from "react-syntax-highlighter/dist/esm/styles/hljs";
+import { Box } from "@mui/material";
+
+type CodeDisplayPropTypes = {
+    text: string;
+    startingLineNumber: number;
+    highlightLine: number;
+};
+
+export default function CodeDisplay(props: CodeDisplayPropTypes) {
+    const lineBackground = (lineNumber: number) => {
+        if (lineNumber == props.highlightLine) {
+            return "#ffff00";
+        }
+    };
+
+    return (
+        <Box sx={{ width: "100%", height: "100%", backgroundColor: "white" }}>
```

Comment:
Move the styling into classes (throughout all files). This is true for both the `Box`, the individual line elements contained in it, and add a class for the highlight line.

Reply:
Got it! Just to confirm, should we be moving these classes to a separate style sheet?

Reply:
Yes, that's correct (you can introduce your own custom stylesheet for this app) 👍